src/xcp_ng_dev/cli.py

```diff
@@ -37,67 +39,111 @@ def is_podman(runner):
         return True
     return False
 
-def main():
-    """ Main entry point. """
-    parser = argparse.ArgumentParser()
+def add_common_args(parser):
     parser.add_argument('-b', '--branch',
                         help='XCP-ng version: 7.6, %s, etc. If not set, '
                              'will default to %s.' % (DEFAULT_BRANCH, DEFAULT_BRANCH))
```

Comment:
It might be a good time to reconsider the optional character of this flag, and drop the `-b`, which would match the `env-create` script.

Reply:
Having a required argument is not exactly pleasant. What is the reason for this change?

Reply:
For comparison, when mock support is added to the tool, users will need to specify the version using a koji tag. IN this case it's easier to see there should be no default since each user wants to build their packages on top of their koji tags, usually

Reply:
> For comparison, when mock support is added to the tool, users will need to specify the version using a koji tag. IN this case it's easier to see there should be no default since each user wants to build their packages on top of their koji tags, usually

I'm not sure of that. I usually build a package with the base and update repositories in koji. I'm quite sure it would be the same with mock.